src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -127,7 +127,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
      * @return {Promise}
      * @resolved {SavedObject}
      */
-    this.init = _.once(() => {
+    this.init = _.once((isDefered) => {
```

Comment:
Yeah, I think that's a good idea. If we want to keep the current code I would probably change the variable to `deferDocSourceRequest` (my second comment was more about naming).

That said, I think all requests to `docSource.fetch()` should probably be using the request queue. I don't think there is any reason to require requesters to opt-into this style of fetching. One wrinkle in that is the interval at which the request queue is cleared. We might need to have some debounced request clearing trigger that gets triggered each time `docSource.fetch()` is called on any request...